src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,32 @@
+$watch-frame-size: 650px;
+$watch-center-point-size: 60px;
+$watch-second-point-size: 60px;
+$second-hand-width: 10px;
+$color-second-hand: rgb(218, 31, 31);
+$round-hand: 60px;
+$minute-hand-speed: 240s;
+$second-hand-speed: 6s;
+
+@keyframes round-hand {
```

Comment:
you have 2 identical `keyframes`, delete one of them

Reply:
they execute  a little bit different animation effects